Dwarf.cpp

```diff
@@ -1,7 +1,23 @@
 #include "Dwarf.h"
 
-//Dwarf::Dwarf
+Dwarf::Dwarf(const std::string& name_, int hp_, int armor_) : Character(hp_, armor_, 4), name(name_)
+{
+    helpfulItems = makeHelpfulItems(2);
+    defensiveItems = makeDefensiveItems(1);
+}
 
-//Dwarf::getName
+void Dwarf::attack(Character& other)
+{
+    std::cout << name << " is attacking " << other.getName() << "!!!" << std::endl;
+    Character::attack(other);
+}
```

Comment:
Remove this function
The base class functionality already does this
- [ ] revise